server-phoenix/lib/helios/developer_user.ex

```diff
@@ -12,7 +12,7 @@ defmodule Helios.DeveloperUser do
     field :git_handle, :string
     field :slack_handle, :string
 
-    timestamps(inserted_at: :created_at, type: :utc_datetime)
+    timestamps(inserted_at: :inserted_at, type: :utc_datetime)
```

Comment:
same as above, also following instances of timestamps.

Reply:
Should be simplified now to `timestamps()` for all relevant files